runtime/ftplugin/man.vim

```diff
@@ -1,41 +1,53 @@
-" Vim filetype plugin file
-" Language:	man
-" Maintainer:	SungHyun Nam <goweol@gmail.com>
-
-if has('vim_starting') && &filetype !=# 'man'
-  finish
-endif
-
-" Only do this when not done yet for this buffer
 if exists('b:did_ftplugin')
   finish
 endif
 let b:did_ftplugin = 1
 
-" Ensure Vim is not recursively invoked (man-db does this)
-" when doing ctrl-[ on a man page reference.
-if exists('$MANPAGER')
-  let $MANPAGER = ''
-endif
+let s:pager = 0
 
-setlocal iskeyword+=\.,-,(,)
-
-setlocal buftype=nofile noswapfile
-setlocal nomodifiable readonly bufhidden=hide nobuflisted tabstop=8
-
-if !exists("g:no_plugin_maps") && !exists("g:no_man_maps")
-  nnoremap <silent> <buffer> <C-]>    :call man#get_page(v:count, expand('<cword>'))<CR>
-  nnoremap <silent> <buffer> <C-T>    :call man#pop_page()<CR>
-  nnoremap <silent> <nowait><buffer>  q <C-W>c
-  if &keywordprg !=# ':Man'
-    nnoremap <silent> <buffer> K      :call man#get_page(v:count, expand('<cword>'))<CR>
+if has('vim_starting')
+  let s:pager = 1
+  " remove all those backspaces
+  silent keeppatterns keepjumps %substitute,.\b,,ge
+  if getline(1) =~# '^\s*$'
+    silent keepjumps 1delete _
+  else
+    keepjumps 1
   endif
+  " I dont think this can ever be perfect.
+  " See `man glDrawArraysInstanced`. Since the title is all caps
+  " it is impossible to tell what the original capitilization was.
+  execute 'file '.'man://'.tolower(matchstr(getline(1), '^\S\+'))
 endif
 
-if exists('g:ft_man_folding_enable') && (g:ft_man_folding_enable == 1)
-  setlocal foldmethod=indent foldnestmax=1 foldenable
+setlocal buftype=nofile
+setlocal noswapfile
+setlocal bufhidden=hide
+setlocal nobuflisted
+setlocal nomodified
+setlocal readonly
+setlocal nomodifiable
+setlocal noexpandtab
+setlocal tabstop=8
+setlocal softtabstop=8
+setlocal shiftwidth=8
+call man#set_window_local_options()
+
+if !exists('g:no_plugin_maps') && !exists('g:no_man_maps')
```

Comment:
Yea, its just convention. the `no_maps` variables are always checked to see if they exist, but other variables are always checked against 1. Weird convention.

Btw, because the diff is outdated, did you not get a notification for https://github.com/neovim/neovim/pull/4449#discussion_r72910313